CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ The following components have had minor internal changes to satisfy the introduc
 * `Dropdown`: Options list is always rendered to the DOM, but is hidden until selected
 * `Textarea` now accepts a new prop `warnOverLimit` to display the character count message in red.
 * Simplify character count in `Textarea`.
+* `Browser`: add a new method `setFocus` to focus on the input field of passed in ref but does not select text
```

Comment:
You've added your release notes to `1.1.0` - which is already released. You should target the next release `1.2.0`